Client/src/App.tsx

```diff
@@ -2,6 +2,8 @@ import React, { Dispatch, SetStateAction, useState } from 'react'
 
 import { IUserAccountModel } from './Remote/Models/IUserAccountModel'
 import NavigationMenu from './Components/NavigationMenu'
+import { SnackbarProvider } from 'notistack'
```

Comment:
We'll have to remember that we need to write a little something on why we are using 3rd party libraries. This one has sweet snack bars for us

Reply:
added to the todo list for release 2